vstsBuild/vstsBuild.psm1

```diff
@@ -17,7 +17,7 @@ function Invoke-VstsPublishBuildArtifact
 {
     $ErrorActionPreference = 'Continue'
     $filter = Join-Path -Path (Get-Destination) -ChildPath '*'
-    Write-VstsMessage -message "Publishing artifacts: $filter"
+    Write-VstsInformation -message "Publishing artifacts: $filter" -type
```

Comment:
`-type` should be removed.

Reply:
This should be fixed, but if you don't find any additional issues, just sign off... I still need to re-test after all these changes.